test-requirements.txt

```diff
@@ -1,3 +1,3 @@
 PyVirtualDisplay==0.1.5
-selenium==2.48.0
+selenium==2.49.0
```

Comment:
Based on the changelog this seems to be small bug fixes (https://github.com/SeleniumHQ/selenium/blob/master/py/CHANGES).  Since the UI tests passed on CircleCI I feel comfortable upping this.